pages/index.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import Home from './index.page';
+
+describe('<Home />', () => {
+  it('should render without crashing', async () => {
+    // Arrange
+    global.fetch = jest.fn();
+
+    jest.spyOn(global, 'fetch').mockResolvedValue({
+      json: () =>
+        Promise.resolve({
+          data: 'url',
+          map: () => {},
+        }),
+      status: 200,
+    } as any);
```

Comment:
[non-blocking] eventually, you can refactor this as a test utility in the `test` folder, since it seems likely that you may with to do this again. There may even be a mocking library for this - I've seen `msw` also used before.

Reply:
I didn't do it this time because I thought it's only currently being used once, and that I could move it later if I need to reuse it. I could investigate mocking libraries in the future, too. I'll bear that in mind. At the moment, I think I wanna do it the 'manual' way just to get a better grip on testing and understanding what's really happening behind the scenes of libraries and such.